runtime/doc/autocmd.txt

```diff
@@ -1417,24 +1417,24 @@ When there is a matching "*Cmd" autocommand, it is assumed it will do the
 writing.  No further writing is done and the other events are not triggered.
 |Cmd-event|
 
-Note that the *WritePost commands should undo any changes to the buffer that
-were caused by the *WritePre commands; otherwise, writing the file will have
+Note that the "*WritePost" commands should undo any changes to the buffer that
+were caused by the "*WritePre" commands; otherwise, writing the file will have
```

Comment:
Even before https://github.com/neovim/tree-sitter-vimdoc/pull/97 these were not rendered correctly: they were treated as `(argument)` which conceals the "*".